src/main/java/CalculatingArea.java

```diff
@@ -0,0 +1,3 @@
+public interface CalculatingArea {
```

Comment:
```suggestion
public interface AreaCalculator{
```

please don't use verbs for class/interface names